tasks/release.py

```diff
@@ -909,6 +919,34 @@ def chase_release_managers(_, version):
         client.chat_postMessage(channel=channel, text=message)
 
 
+@task
+def chase_for_qa_cards(_, version):
+    from collections import defaultdict
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏For such 'small' imports couldn't we do it at the start of the file instead ?